src/gui/mzroll/background_peaks_update.cpp

```diff
@@ -452,5 +452,40 @@ void BackgroundPeakUpdate::updateGroups(QList<PeakGroup> &groups,
             }
         }
         group.groupStatistics();
+        group.children.clear();
+    }
+
+    MavenParameters* mavenparameters = new MavenParameters();
+    mavenparameters = mavenParameters;
+    mavenparameters->allgroups.clear();
+    for (PeakGroup& group : groups) {
+        mavenparameters->allgroups.push_back(group);
+    }
+
+    if (mavenparameters->pullIsotopesFlag) {
+        for (PeakGroup& group : groups) {
+            if (!group.isIsotope())
+            {
+                bool C13Flag = mavenparameters->C13Labeled_BPE;
+                bool N15Flag = mavenparameters->N15Labeled_BPE;
+                bool S34Flag = mavenparameters->S34Labeled_BPE;
+                bool D2Flag = mavenparameters->D2Labeled_BPE;
+
+                IsotopeDetection::IsotopeDetectionType isoType;
+                isoType = IsotopeDetection::PeakDetection;
+
+                IsotopeDetection isotopeDetection(
+                    mavenparameters,
+                    isoType,
+                    C13Flag,
+                    N15Flag,
+                    S34Flag,
+                    D2Flag);
+                isotopeDetection.pullIsotopes(&group);
+                for (PeakGroup& child : group.children) {
+                    child.setTableName(group.tableName());
+                }
+            }
+        }
```

Comment:
will make this change

Reply:
Right. I tried this myself. Didn't work. It is because of the way isotopic group are created is different enough. Nearest peaks, RT bounds check and all that would have to happen again. Alright then, we can stick with this solution.

One bug: RTs for groups in bookmark table are not recalculated.